apps/src/applab/setPropertyDropdown.js

```diff
@@ -225,9 +234,12 @@ module.exports.getInternalPropertyInfo = function (element, friendlyPropName) {
  *   types, provides full list of properties across all types.
  */
 module.exports.setPropertyDropdown = function () {
-  return function () {
+  return function (editor) {
     // Note: We depend on "this" being the droplet socket.
```

Comment:
This comment is a little confusing now, since `this` won't be a socket in text mode.  But that logic check doesn't happen until `this.parent` is passed and `!block` checked above.